css/style.css

```diff
@@ -543,5 +1031,236 @@
+    font-size: 12px;
+    line-height: 15px;
+    font-weight: lighter;
+  }
+
+  .fruture-text-description-wifi {
+    margin: 0;
+    max-width: 150px;
+    font-size: 20px;
+    line-height: 24px;
+    font-weight: normal;
+  }
+  
+  .future-drive-list__item:nth-child(1) {
+    margin-bottom: 20px;
+    padding-left: 60px;
+    background-image: url(/img/screen.svg);
+    background-repeat: no-repeat;
+    background-position-y: 5px;
+  }
+
+  .future-drive-list__item:nth-child(2) {
+    margin-bottom: 20px;
+    padding-left: 60px;
+    background-image: url(/img/wifi.svg);
+    background-repeat: no-repeat;
+    background-position-y: 20px;
+  }
+
+  .future-drive-list__item:nth-child(3) {
+    margin-bottom: 20px;
+    padding-left: 60px;
+    background-image: url(/img/bag.svg);
+    background-repeat: no-repeat;
+    background-position-y: 15px;
+  }
+
+  .article-footer {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 20px;
+    padding-left: 30px;
+    font-size: 13px;
+    line-height: 16px;
+    text-transform: uppercase;
+    font-weight: bold;
+    background-image: url(/img/i.svg);
+    background-repeat: no-repeat;
+    background-position-y: 17px;
+  }
+
+  .text-info {
+    margin-right: 15px;
+    padding-top: 7px;
+  }
+
+  .order-button-white {
+    padding: 15px 38px;
+    color: #fff;
+    background-color: #e51937;
+    border: 2px solid #FFF;
+  }
+
+  .safe-car {
+    position: relative;
+    display: flex;
+  }
+
+  .safe-car-article {
+    min-width: 440px;
+    background-color: #e51937;
+    color: #fff;
+  }
+
+  .article-text {
+    margin-bottom: 50px;
+  }
+
+  .safe-list {
+    margin: 0;
+    margin-bottom: 50px;
+    padding: 0;
+    list-style-type: none;
+  }
+
+  .second-title {
+    margin-bottom: 50px;
+  }
+
+  .safe-list__item {
+    display: flex;
+    margin-bottom: 15px;
+    font-size: 17px;
+    line-height: 21px;
+    font-weight: bold;
+    text-transform: uppercase;
+  }
+
+  .safe-list-text {
+    margin: 0;
+    padding-top: 5px;
+    width: 120px;
+  }
+
+  .model-img {
+    position: absolute;
+    top: 120px;
+    left: 650px;
+  }
+
+  .model-img-car {
+    position: absolute;
+    height: auto;
+    width: 75%;
+    top: 140px;
+    left: 350px;
+  }
+
+  .container-safe-star {
+    margin-bottom: 50px;
+    background-image: url(/img/security.svg);
+    background-repeat: no-repeat;
+    background-position-y: 10px;
+  }
+
+  .safe-text {
+    margin: 0;
+    padding-bottom: 0;
+    padding-left: 80px;
+    font-size: 50px;
+    line-height: 61px;
+    font-weight: bold;
+  }
+
+  .safe-text-description {
+    margin: 0;
+    padding-left: 80px;
+    font-size: 25px;
+    line-height: 30px;
+    font-weight: medium;
+  }
+
+  .article-footer-text {
+    margin-right: 10px;
+    padding-top: 7px;
+    text-transform: uppercase;
+  }
+
+  .article-footer {
+    margin-bottom: 40px;
+  }
+
+  .footer-link {
+    margin-left: auto;
+    align-self: flex-end;
+  }
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6120148)